test/TestExtensions/DefaultClusterFixture.cs

```diff
@@ -8,8 +8,11 @@ public class DefaultClusterFixture : BaseTestClusterFixture
         protected override TestCluster CreateTestCluster()
         {
             var options = new TestClusterOptions();
+#if !NETSTANDARD_TODO
+            // waiting for OrleansProviders project to be ported over to vNext
             options.ClusterConfiguration.AddMemoryStorageProvider("Default");
             options.ClusterConfiguration.AddMemoryStorageProvider("MemoryStore");
+#endif
```

Comment:
well, the point is that #if is required to get it to build. This comment was just a heads up to know how to resolve conflicts, not to let git be smart about it and autofix it (otherwise there would be  no need for a github comment :P )